### DISCUSSION TITLE
Migration support

### DISCUSSION THREAD

**Discussion by rsignavong** [rbatis/rbatis#279] [General]
Hello,

What do you use or recommend for database migrations?
Do you plan to add it?

**Comment by zhuxiujia** [comment]
Because rbatis relies on sqlx-core at the bottom, you can use 'rbatis core' driver or 'sqlx-core' driver migration。
However, in the future, I will provide a graphical UI interface

**Comment by debarshri** [comment]
Could you create an example how to do it? I love everything about the project, but I think if you improve/give examples for migrations the project will be very very useful.

**Comment by conanchen** [comment]
How to do like diesel  ?    

[The diesel_migrations crate](https://diesel.rs/guides/getting-started) provides the embed_migrations! macro, allowing you to embed migration scripts in the final binary. Once your code uses it, you can simply include embedded_migrations::run(&db_conn) at the start of your main function to run migrations every time the application starts.

**Comment by 1nF0rmed** [comment]
Any update or documentation we can look at @zhuxiujia ?
Thank you

**Comment by msmaiaa** [comment]
It would be good to have a built-in migration system so i don't have to rely on third party crates

**Comment by mystique09** [comment]
Will this still be supported? It would be nice to add migration support.
